cub/test/catch2_test_temporary_storage_layout.cu

```diff
@@ -31,8 +31,10 @@
 
 #include <memory>
 
+#include "catch2_test_helper.h"
 #include "cub/detail/temporary_storage.cuh"
-#include "test_util.h"
+
+using values = c2h::enum_type_list<int, 1, 4, 42>;
 
 template <int Items>
 std::size_t GetTemporaryStorageSize(std::size_t (&sizes)[Items])
```

Comment:
nit: In tests and details namespace we try to stick to `snake_case`, except for template parameters, for which we use `CamelCase`.